packages/block-editor/src/hooks/block-bindings.scss

```diff
@@ -4,11 +4,3 @@ div.block-editor-bindings__panel {
 		color: inherit;
 	}
 }
-
-.block-editor-bindings__popover {
```

Comment:
We may be able to remove the `block-editor-bindings__popover` className from the `DropdownMenuV2` instance altogether as well.

Reply:
Good point — done!